packages/mint-components/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@saasquatch/mint-components",
   "title": "Mint Components",
-  "version": "1.6.2",
+  "version": "1.7.0-14",
```

Comment:
This probably needs to be a 1.6.x level change. This is because our microsites all have `1.6.x` as the mint version, meaning for this to be backwards compatible without changing all the versions, we need this to be a patch level change